src/consensus/aft/raft.h

```diff
@@ -336,13 +336,17 @@ namespace aft
       return {get_term_internal(state->commit_idx), state->commit_idx};
     }
 
-    std::optional<std::pair<Term, Index>> get_signable_commit_term_and_idx()
+    std::optional<kv::Consensus::SignableTXIndices>
+    get_signable_commit_term_and_idx()
     {
       std::lock_guard<SpinLock> guard(state->lock);
       if (state->commit_idx >= election_index)
       {
-        return std::pair<Term, Index>{get_term_internal(state->commit_idx),
-                                      state->commit_idx};
+        kv::Consensus::SignableTXIndices r;
```

Comment:
We use Tx everywhere I believe, so this should probably be a SignableTxIndices, technically.